usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -44,7 +47,7 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
     if is_sle_micro():
         pytest.skip('Skipping SLE Micro')
 
-    if not setup_swap(host):
+    if not setup_swap(host, btrfs=is_sle_micro()):
```

Comment:
I don't understand according lines 47-48 we suppose to skip this test so if SLM we would simply not get here :thinking:

Reply:
> I don't understand according lines 47-48 we suppose to skip this test so if SLM we would simply not get here 🤔

True, but I added the needed code just for correctness.  I can either remove the skip before, remove the commit, or leave as it is.